.devcontainer/scripts/setup-local-settings.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/bash
+set -e
+
+WORKFLOW_TEMPLATE_SETTINGS="./source/workflow-designtime/local.template.settings.json"
+WORKFLOW_LOCAL_SETTINGS="./source/workflow-designtime/local.settings.json"
+
+HOST_TEMPLATE_SETTINGS="./source/local.template.settings.json"
+HOST_LOCAL_SETTINGS="./source/local.settings.json"
+
+# Check if the file exists
+if [ ! -f "$WORKFLOW_LOCAL_SETTINGS" ]; then
```

Comment:
nit: I'm not a fan of `if not x` with an else as it generally adds an extra cognitive step for reasoning about the else condition (`not not x`)